README.md

```diff
@@ -1,2 +1,52 @@
-# js-testing-practice
-This project is JavaScript testing practice.
+# Js-testing-practice
+
+## Description
+
+[Odin] TDD (Test Driven Development) Js-testing-practice using Jest.
+
+This repository contains multiple functions separated each within its folder containing two files: the module `function.js` and a Jest test file `function.test.js`.
+
+The main idea is simply to start working on your code by writing automated tests before writing the code that is being tested.
+
+## Setup
+
+- Clone the repository on your local machine
+- cd into the folder, find the index.html file and open it
+- npm install
+- npm test
+
```

Comment:
- Write the Actual commands to clone <including the clone URL>, as well as the one to `cd` into directory
- Also put the commands to appear as commands by putting them into backtits characters (so that they cn appear as this --> `code`)